### DISCUSSION TITLE
What port are vercel projects servered on?

### DISCUSSION THREAD

**Discussion by Milo123459** [vercel/vercel#5914] [Help] [answered]
Hi, I have a next app where I have to fetch something from the API but I don't know the port it is running on. Is there a way to find which port the application is running on?

**Comment by styfle** [comment, marked as answer]
Hi @Milo123459 

If your Next.js application is using using [API Routes](https://nextjs.org/docs/api-routes/introduction) you can do one of these:

1. Move the shared logic into a separate file and import that module in `/api` as well as any `/pages` that need the data
2. Or you could call `fetch('/api/my-function')` on the client-side only

No need to use ports 🙂
